examples/forecast_evaluation/reformat_airnow_rapchemtest.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
```

Comment:
I suggest renaming the file to just reformat_airnow.py, as it is not specific to rapchem ...

Reply:
Good point David, but given our discussion in the last meeting, let's keep all the RAP-chem specific files in this directory as is and you and Zach can work on creating generalized versions to include in the main code as a separate pull request.